src/NuGet.Clients/NuGet.CommandLine/Commands/SignCommand.cs

```diff
@@ -0,0 +1,190 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using System.Security;
+using System.Security.Cryptography;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+using System.Threading.Tasks;
+using NuGet.Commands;
+using NuGet.Packaging.Signing;
+using NuGet.Shared;
+
+namespace NuGet.CommandLine
+{
+    [Command(typeof(NuGetCommand), "sign", "SignCommandDescription",
+        MinArgs = 1,
+        MaxArgs = 1,
+        UsageSummaryResourceName = "SignCommandUsageSummary",
+        UsageExampleResourceName = "SignCommandUsageExamples",
+        UsageDescriptionResourceName = "SignCommandUsageDescription")]
+    public class SignCommand : Command
+    {
+        // Default constructor used only for testing, since the Command Default Constructor is protected
+        public SignCommand() : base()
+        {
+        }
+
+        public ISignCommandRunner SignCommandRunner { get; set; }
```

Comment:
It seems like it would be easier if we had a static helper that created SignArgs and passed it back instead of needing to catch the SignArgs result by passing a mock SignCommandRunner in to get that result.